src/transformState.js

```diff
@@ -6,6 +6,34 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const elem of actions) {
```

Comment:
use destructurization in future
```suggestion
  for (const { type, extraData, ... } of actions) {
```